docs/MPQEditor.md

```diff
@@ -0,0 +1,63 @@
+# MixedPrecisionQantizationEditor (MPQEditor for short)
+
+## About
+
+MPQEditor povides easy way of creation/editing MPQ json files, suitable for Quantization step in CfgEditor. Also it gives the ability to select the most sensitive to quantization layers using `visq` data. 
```

Comment:
plz split this and below lines to match 80 or 100 cols.

Reply:
ok